src/js/polyfills/webvtt.js

```diff
@@ -743,7 +749,7 @@ function moveBoxToLinePosition(window, styleBox, containerBox, boxPositions, num
         // need to increase our initial position by the length or width of the
         // video, depending on the writing direction, and reverse our axis directions.
         if (linePos < 0) {
-            position += cue.vertical === '' ? containerBox.height : containerBox.width;
+            position += !cue.vertical ? containerBox.height : containerBox.width;
```

Comment:
Can you make this a positive conditional? `cue.vertical ? containerBox.width : containerBox.height`